test/cpp/qps/qps_test.cc

```diff
@@ -64,16 +67,23 @@ static void RunQPS() {
   const auto result =
       RunScenario(client_config, 1, server_config, 1, WARMUP, BENCHMARK, -2);
 
-  ReportQPSPerCore(result, server_config);
-  ReportLatency(result);
+  std::set<ReportType> types;
+  types.insert(grpc::testing::ReportType::REPORT_QPS_PER_CORE);
+  types.insert(grpc::testing::ReportType::REPORT_LATENCY);
+  for (const auto& reporter : reporters) {
+    reporter->Report({client_config, server_config, result}, types);
```

Comment:
Sorry, some old compilers that we support don't like this kind of brace initialization.